tests/test_cli.py

```diff
@@ -199,6 +199,27 @@ def test_cli_predict_point_crs(mocker, local_model_path):  # noqa: F811
     assert "Invalid value for '--crs" in result.stdout
 
 
+def test_graceful_exit_when_no_satellite_data():
+    # use location and date in SF where it was cloudy all month and there is no valid imagery
+    result = runner.invoke(
+        app,
+        [
+            "predict-point",
+            "--date",
+            "2024-01-04",
```

Comment:
Are we sure that this won't change at some point -- it is possible more imagery will be added since this is so recent?

Reply:
There's not typically a delay in when sentinel 2 imagery is made available so this should reflect the final available imagery. If for some reason this changes, it will get caught in the weekly test runs and we can push a fix (e.g. mock the query response from planetary computer)